core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -2212,9 +2212,9 @@ private[spark] object Utils extends Logging {
    * incompatible. In environments where dynamic allocation is turned on by default,
    * the latter should override the former (SPARK-9092).
    */
-  def isDynamicAllocationEnabled(conf: SparkConf): Boolean = {
+  def isDynamicAllocationEnabled(conf: SparkConf, isLocal: Boolean): Boolean = {
```

Comment:
Can we add a default value `isLocal: Boolean = false` to avoid changing several other parts?